src/root/components/conteneur/conteneur.component.ts

```diff
@@ -10,7 +10,16 @@ const ANIMATION_DURATION = 300 as const;
   selector: 'app-conteneur',
   templateUrl: './conteneur.component.html'
 })
-export class ConteneurComponent {
+export class ConteneurComponent implements OnInit {
+  ngOnInit() {
+    this.router.events.pipe(filter((event) => event instanceof NavigationEnd)).subscribe(() => {
+      let div = document.getElementById('conteneurDiv');
```

Comment:
Ne pas utiliser `getElementById`, mais plutôt des [ElementRef](https://stackoverflow.com/questions/32693061/how-can-i-select-an-element-in-a-component-template)